src/main/java/com/github/manolo8/darkbot/gui/tree/editors/ImageOpener.java

```diff
@@ -0,0 +1,72 @@
+package com.github.manolo8.darkbot.gui.tree.editors;
+
+import com.github.manolo8.darkbot.config.ImageWrapper;
+import com.github.manolo8.darkbot.gui.AdvancedConfig;
+import eu.darkbot.api.config.util.OptionEditor;
+import com.github.manolo8.darkbot.gui.utils.Strings;
+import eu.darkbot.api.config.ConfigSetting;
+
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.awt.*;
+import java.io.File;
+
+public class ImageOpener extends JButton implements OptionEditor<ImageWrapper> {
```

Comment:
```suggestion
public class ImagePicker extends JButton implements OptionEditor<ImageWrapper> {
```
I think image picker is a better suited name for this